src/abi/balancerBatchRouter.ts

```diff
@@ -1,16 +1,8 @@
 export const balancerBatchRouterAbi = [
     {
         inputs: [
-            {
-                internalType: 'contract IVault',
-                name: 'vault',
-                type: 'address',
-            },
-            {
-                internalType: 'contract IWETH',
-                name: 'weth',
-                type: 'address',
-            },
+            { internalType: 'contract IVault', name: 'vault', type: 'address' },
+            { internalType: 'contract IWETH', name: 'weth', type: 'address' },
```

Comment:
I see that most changes are related to formatting. 🤔 
I believe this is being auto-formatted with Biome, but if that's the case, then why it wasn't already formatted before?
Anyway - let's hope it doesn't change again next time we update the ABI 😅

Reply:
Initially my IDE refused to format because it couldn't find Biome, but it started working after I installed the biomejs extension for vscode

I'll make note of this in the SDK Dev Guide on notion

Reply:
On that topic, since most people have Prettier extension installed on vscode, maybe it's also nice to mention that you have to disable it within the workspace for Biome extension to work as expected, otherwise they will conflict.

Reply:
Agree with sentiment, but I don't think I had to disable my Prettier extension that is setup to format on save

I'm not too familiar with formatters and linters. I'll investigate a bit more